logger/logger.go

```diff
@@ -392,6 +393,43 @@ func WithTime(t time.Time) *log.Entry {
 	return log.WithTime(t)
 }
 
+// HTTPLogger : wrapper for http logging
+func HTTPLogger(inner http.Handler, name string) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		sourced().Infof(
+			">>>>> %s\t%s\t%s",
```

Comment:
The logrus package does not expand escape characters (e.g. \t).  Instead, it shows up like this in the log file:

`time="2019-08-19T11:13:11-07:00" level=info msg=">>>>> GET\t/api/v1/initiators\tHostInitiators" file="server.go:1947"`

For improved readability, we could change the format to something like this:

`">>>>> %s %s - %s",`

Generates output like this:

`time="2019-08-19T11:15:13-07:00" level=info msg=">>>>> GET /api/v1/initiators - HostInitiators" file="server.go:1947"`

Same comment for the logged exit event as well.  If you think the output file could be parsed, and the tab escape character is helpful, disregard comment.

Also, since entry/exit routines are typically trace logs, and not info logs, should these be sourced().Tracef()?

Reply:
sure, let me fix that..